### PR TITLE
[DB-11889]: Fixed Sharding recommendations application to Oracle exported schema

### PR DESCRIPTION
**Pull request by sanyamsinghal**
- case-sensitivity handled correctly and valid comparison of object names are made

fixes https://yugabyte.atlassian.net/browse/DB-11889

### CODE REVIEW

**Comment by sanyamsinghal** [comment]
Jenkins Pipeline(PG and Oracle tests) - https://jenkins.dev.yugabyte.com/job/users/job/yb-voyager-testing/job/yb-voyager-testing-pipeline/2787/

File: yb-voyager/cmd/exportSchema.go [hunk 1]
```diff
@@ -378,12 +380,33 @@ func applyShardingRecommendationIfMatching(sqlInfo *sqlInfo, shardedTables []str
...
+		relation.Schemaname+"."+relation.Relname)
+
+	match := false
+	switch source.DBType {
+	case POSTGRESQL:
+		match = slices.Contains(shardedTables, parsedTableName)
+	case ORACLE:
+		// TODO: handle case-sensitivity properly
+		for _, shardedTable := range shardedTables {
+			parts := strings.Split(shardedTable, ".")
```
Comment (priyanshi-yb) [inline yb-voyager/cmd/exportSchema.go:396]: How will reserved words table names are stored in the `shardedTable`?

Comment (sanyamsinghal) [reply yb-voyager/cmd/exportSchema.go:396]: shardedTable is the information we fetch from the source db first.
There we fetch from system catalog tables. In which the names are stored in a case sensitive way but without any quoting.

For example:
1. Normal table as per oracle format will be stored as: `EMPLOYEES`
2. Case sensitive table will be stored as this: `EmployEEs`
3. Reserved words table will be stored as case 2

Comment (priyanshi-yb) [reply yb-voyager/cmd/exportSchema.go:396]: Okay, so just to confirm pg parser always gives unquoted name right?

**Review by priyanshi-yb — APPROVED** [review]

File: yb-voyager/cmd/exportSchema.go (hunk 1 above)
Comment (sanyamsinghal) [reply yb-voyager/cmd/exportSchema.go:396]: yes, i tested the code with case-sensitivity-reserved-words in oracle and it passed.

**Pull request merged**